CharacterCreation/01_00_Species_Overview.txt

```diff
@@ -36,6 +36,22 @@ Species Overview
 
 ===== Humans =====
 
+    Humans are bipedal organic mammals. They origionally hail from a formerly
+blue-green planet called Earth, and now make up the majority of the Terran 
+Empire. Humans are hearty, enterprising and resourceful. 
+
+    Humans can excell at any class. You could be a grizzled human soldier who's
+seen more than their fair share of combat. Or you could be the charismatic 
+gunslinging galatic bounty hunter who always gets their man, but rarely gets the
+girl. You could be the intuitive human engineer who takes care of her drones 
+like they were her beloved pets. Or the recent med student grad who's a whiz
```

Comment:
Gr: "are her beloved pets, or the..."
Suggestion: "med school" or "recently graduated doctor"